sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -881,6 +881,16 @@ object SQLConf {
       .intConf
       .createWithDefault(10000)
 
+  val HIVE_STYLE_PARTITION_OVERWRITE =
+    buildConf("spark.sql.hiveStylePartitionOverwrite")
```

Comment:
i wouldn't call it like this. I'd actually describe what it does, tableOverwrite vs partitionOverwrite.